focus-ios/Blockzilla/AppDelegate.swift

```diff
@@ -111,6 +111,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             if let bundleID = Bundle.main.bundleIdentifier {
                 UserDefaults.standard.removePersistentDomain(forName: bundleID)
             }
+            UIView.setAnimationsEnabled(false)
```

Comment:
I would like to assume that the animations are not needed when the app is run in a test environment. No new launch argument is needed.